build.gradle.kts

```diff
@@ -59,6 +67,12 @@ dependencies {
     implementation("org.litote.kmongo:kmongo:${Version.KMONGO}")
 
     implementation("com.willowtreeapps.assertk:assertk:${Version.ASSERTK}")
+
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
```

Comment:
Das kann noch raus (steht schon in Zeile 45), will nur nich in deinem branch rum mehren während du was machst :D

Reply:
done